lib/matplotlib/colors.py

```diff
@@ -243,6 +243,17 @@ def _check_color_like(**kwargs):
             raise ValueError(f"{v!r} is not a valid value for {k}")
 
 
+def check_color_like_list(**kwargs):
+    """
+    For each *key, lst* pair in *kwargs*, check that every
+    element *v* in *lst* is color-like.
+    """
+    for k, lst in kwargs.items():
+        invalid_col = list((lambda v: not is_color_like(v), lst))
```

Comment:
I think this is not correct. Are you intending to use `map()`? Anyway, I think a list comprehension is the simplest solution here:
```suggestion
        invalid_col = [c for c in lst if not is color_like(c)]
```

Reply:
Ideally yes. Though I would not strictly require this from a first-time contributor and a relatively simple function - it adds to the complexity of getting a first PR done. We can always add a test ourselves after merging.

@i-deal are you comfortable writing a test? (If not, that's fine for a first contribution. We will do it afterwards).

Reply:
I'm not sure that I can write an adequate test for these functions as I'm not familiar with the testing module.